src/openforms/registrations/contrib/zgw_apis/api/views.py

```diff
@@ -3,35 +3,59 @@
 from drf_spectacular.utils import extend_schema, extend_schema_view
 from rest_framework import authentication, permissions
 from rest_framework.views import APIView
+from zds_client import Client
 from zgw_consumers.api_models.base import factory
 from zgw_consumers.api_models.catalogi import Catalogus, InformatieObjectType
 from zgw_consumers.service import get_paginated_results
 
 from openforms.api.views import ListMixin
 from openforms.registrations.contrib.zgw_apis.models import ZgwConfig
 
+from ...objects_api.models import ObjectsAPIConfig
```

Comment:
cross-app relative import, but the fact that this code is now relevant for both ZGW and Objects API makes me think the view itself should rather be moved to `openforms.contrib.catalogi` or something like that